test/views/payments/NoPaymentsViewSpec.scala

```diff
@@ -138,4 +139,34 @@ class NoPaymentsViewSpec extends ViewBaseSpec {
       }
     }
   }
+
+  "Rendering the open payments page" should {
+
+    "display the covid message" when {
+
+      "the display covid feature switch is on" in {
+        mockConfig.features.displayCovidMessage(true)
+
+        lazy val view = views.html.payments.noPayments(user, hasDirectDebit = Some(true))
+        lazy implicit val document: Document = Jsoup.parse(view.body)
+
+        document.select(Selectors.covidPartial).toString should include("Coronavirus (COVID 19) VAT deferral")
+      }
+
+    }
+
+    "not display the covid message" when {
+
+      "the display covid feature switch is off" in {
+        mockConfig.features.displayCovidMessage(false)
+
+        lazy val view = views.html.payments.noPayments(user, hasDirectDebit = Some(true))
+        lazy implicit val document: Document = Jsoup.parse(view.body)
+
+        document.select(Selectors.covidPartial).toString shouldNot include("Coronavirus (COVID 19) VAT deferral")
```

Comment:
There's a method in ViewBaseSpec called 'elementExtinct' which I think might be better.